hooks/useSWRFetch.js

```diff
@@ -0,0 +1,9 @@
+import useSWR from "swr";
+
```

Comment:
Looks like you created your hook just to have your `fetcher` function available. Do you know about [SWRConfig](https://swr.vercel.app/docs/global-configuration)? Using this module, you can get rid of your own hook and use the `useSWR` hook directly

Reply:
Oh you're right! Complete forgot about that. I'll fix it immediately. Thanks.